src/main/java/internet/shop/dao/impl/CarDaoImpl.java

```diff
@@ -1,54 +1,243 @@
+import internet.shop.model.Manufacturer;
+import internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
 @Dao
 public class CarDaoImpl implements CarDao {
     @Override
     public Car create(Car car) {
-        Storage.addCar(car);
+        String queryForCar = "INSERT INTO cars (manufacturer_id, model) values (?, ?);";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatementOfCar = connection.prepareStatement(queryForCar,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatementOfCar.setLong(1, car.getManufacturer().getId());
+            preparedStatementOfCar.setString(2, car.getModel());
+            preparedStatementOfCar.executeUpdate();
+
+            ResultSet generatedKeys = preparedStatementOfCar.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't create the data - "
+                    + car + " into cars", exception);
+        }
         return car;
     }
 
     @Override
     public Optional<Car> get(Long id) {
-        return Storage.getCars().stream()
-                .filter(x -> x.getId().equals(id))
-                .findFirst();
+        String queryCreateCarAndManufacturer = "select c.id as car_id, c.model "
+                + "as car_model, m.id as manufacturer_id, m.name as manufacturer_name, "
+                + "m.country as manufacturer_country from cars c "
+                + "inner join manufacturers m on m.id = c.manufacturer_id "
+                + "where c.deleted = false and m.deleted = false and c.id = ?;";
+
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement stmtCarManufacturer = connection
+                         .prepareStatement(queryCreateCarAndManufacturer)) {
+            stmtCarManufacturer.setLong(1, id);
+            ResultSet resultSetCar = stmtCarManufacturer.executeQuery();
+            if (resultSetCar.next()) {
+                car = createCar(resultSetCar);
+                Manufacturer manufacturer = createManufacturer(resultSetCar);
+                car.setManufacturer(manufacturer);
+            }
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't get the data by id: " + id, exception);
+        }
+        if (car != null) {
+            List<Driver> drivers = getDriversByCarId(car.getId());
+            for (Driver driver: drivers) {
+                car.setDrivers(driver);
```

Comment:
```suggestion
                car.addDriver(driver);
```
just rename the method in Car class

Reply:
done